project/SparkRedshiftBuild.scala

```diff
@@ -55,6 +55,8 @@ object SparkRedshiftBuild extends Build {
       resolvers +=
         "Spark 1.5.0 RC3 Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1143",
       libraryDependencies ++= Seq(
+        // The SLF4J API is provided by Spark:
+        "org.slf4j" % "slf4j-api" % "1.7.5" % "provided",
```

Comment:
This probably doesn't matter since this library will always be included with Spark, but I believe you are actually supposed to include this jar.  That way it gracefully fails even if the user has not included a specific slf4j binding.